src/engraving/dom/segment.h

```diff
@@ -50,9 +50,9 @@ enum class SegmentType {
     ClefStartRepeatAnnounce    = 0x40,
     KeySigStartRepeatAnnounce  = 0x80,
     TimeSigStartRepeatAnnounce = 0x100,
-    Clef                  = 0x200,
-    BarLine               = 0x400,
-    Breath                = 0x800,
+    Breath                = 0x200,
```

Comment:
You beat me to it. Though I'm pretty sure that breaths should also go before all the "annouce" segments, right?